File: servicio-orquestador/src/main/java/co/edu/unicauca/orquestador/fachadaServices/DTORespuesta/RespuestaConsultaPazySalvoFinancieraDTO.java
package co.edu.unicauca.orquestador.fachadaServices.DTORespuesta;

import java.time.LocalDate;

import lombok.Data;

@Data
public class RespuestaConsultaPazySalvoFinancieraDTO {
    private double montoAdeudado;
    private String motivoDeuda;
    private LocalDate fechaGeneracionDeuda;
    private LocalDate fechaLimitePago;
    private String estadoDeuda;
}

File: app-estudiante/src/app/modelo/RespuestaConsultaPazySalvoFinancieraDTO.ts
export class RespuestaConsultaPazySalvoFinancieraDTO {
  montoAdeudado!: number;
  motivoDeuda!: string;
  fechaGeneracionDeuda!: string;  // formato 'YYYY-MM-DD'
  fechaLimitePago!: string;       // formato 'YYYY-MM-DD'
  estadoDeuda!: string;
}